src/admin.js

```diff
@@ -1,14 +1,26 @@
 /* global window, fetch */
 /* eslint no-labels: ['error', { 'allowLoop': true }] */
 import 'fetch-everywhere';
-import StitchClient from './client';
+import { StitchClient } from './client';
 import ADMIN_CLIENT_TYPE from './common';
 import { ADMIN_CLIENT_CODEC } from './auth/common';
 
 const v2 = 2;
 const v3 = 3;
 
-export default class Admin extends StitchClient {
+export class StitchAdminFactory {
+  constructor() {
```

Comment:
[q] We don't even need this here right? They can totally make a factory but it'd be useless